al_servicec.js

```diff
@@ -213,7 +213,13 @@ class AzcollectC extends AlServiceC {
        let statusBody = {
             cf_stack_name : registrationValues.stackName,
             version : registrationValues.version,
-            collect_rule: registrationValues.collectRule
+
+            //CWE
+            collect_rule: registrationValues.collectRule,
```

Comment:
@alexturkin let's add `custom_fields` array into `registrationValues` and just merge/append them into `statusBody` in order to avoid changing this file for every collector type.